docs/make.jl

```diff
@@ -1,3 +1,4 @@
+push!(LOAD_PATH,"../src/")
```

Comment:
I also think this can be removed.  In Julia the times when you need to modify `LOAD_PATH` are very rare.

Reply:
will remove. was added to create docs locally on Windows.